ada_url/ada_adapter.py

```diff
@@ -18,7 +18,8 @@
 
 GET_ATTRIBUTES = frozenset(PARSE_ATTRIBUTES)
 SET_ATTRIBUTES = frozenset(URL_ATTRIBUTES)
-DELETE_ATTRIBUTES = frozenset(('port', 'hash', 'pathname', 'search'))
+CLEAR_ATTRIBUTES = frozenset(('port', 'hash', 'search'))
```

Comment:
Would you mind adding a comment for these attributes? It's not clear (at least to me) what the difference between UNSET and CLEAR is.

Reply:
Added explanatory comments in 6039a8096c6145192dbd8b14db5566f95a7654a9